.github/workflows/tail-call.yml

```diff
@@ -2,12 +2,14 @@ name: Tail calling interpreter
 on:
   pull_request:
     paths:
+      - '.github/workflows/tail-call.yml'      
```

Comment:
This feels wrong to me, but if it's needed, then sure. Hopefully someone else can sign off on this part of the change.